### DISCUSSION TITLE
Better JEI compat for plated food

### DISCUSSION THREAD

**Discussion by drakray** [vectorwing/FarmersDelight#429] [Ideas]
A comment I got from players on my private server:
When looking at food by value in JEI, plated food show up, but you can't easily get the recipe since they are obtained by bowl-clicking on block food.
A new JEI tab would be very useful to be able to retro-engineer those food, and would be an investment for future use of the function, should you or some add-on add more block food.

Thank you for your good work :D

**Comment by MortisLegati** [comment]
I have _no idea_ what the various cooking pot recipes are because they're not stated anywhere and you can't use a recipe browser for it.
